src/test/java/com/google/javascript/gents/singleTests/comments.js

```diff
@@ -40,7 +40,7 @@ var foo = function(deleted, notdeleted) { return deleted + notdeleted; };
  * @package {number}
  * @const {number}
  */
-var x;
+var x = 1;
```

Comment:
As long as there is a test for each of these being removed, I don't think it matters if they are on different variables.